bin/test-release.sh

```diff
@@ -6,12 +6,10 @@ argumentsRest=${@:2}
 suffix=${argumentsRest:-}
 
 coursier fetch \
-  org.scalameta:mtags_3.0.0-M1:$version \
-  org.scalameta:mtags_3.0.0-M2:$version \
-  org.scalameta:mtags_3.0.0-M3:$version \
   org.scalameta:mtags_3.0.0-RC1:$version \
   org.scalameta:mtags_3.0.0-RC2:$version \
   org.scalameta:mtags_3.0.0-RC3:$version \
+  org.scalameta:mtags_3.0.0:$version \
```

Comment:
Sort of random, but are Scala 3 artifacts supposed to be `_3` or `_3.0.0`? This morning I hit on this in coursier and we had to correct to `_3` https://github.com/coursier/apps/pull/98

Reply:
Mtags always resolve with the full scala version similar to anything that depend on the compiler. Just like the other mtags versions. This is finally an end to the very confusing time 😅

Reply:
ahhh of course, duh. sorry wasn't thinking at all there.